### ISSUE TITLE
Street View Rendered Broken

### ISSUE THREAD

**Issue by willy-juisan** [GoogleWebComponents/google-map#396]
If I tried to open street view by dropping the pegman anywhere in the map, it will automatically open the street view panorama. Strange thing is, some control are rendered broken, e.g. addressControl (rendered as text), closeButton (not rendered), panControl (rendered as text). We could check this problem using the demo at [webcomponents.org](https://www.webcomponents.org/element/GoogleWebComponents/google-map).
But when I tried to use the vanilla Google Maps Javascript API to create a map, and open the street view, it will not produce this problem. We could try the demo at [HERE](https://developers.google.com/maps/documentation/javascript/examples/map-simple). 

How do I fix this problem? What is the root cause of this problem? Thanks and regards.

**Comment by petearvanitis** [comment]
I'm seeing the same issue. Was there ever a workaround/fix discovered?

**Comment by willy-juisan** [comment]
No, I haven't found any workaround / fix yet. Please @e111077 kindly take a look at this problem. Thank you.
